src/instructlab/model/chat.py

```diff
@@ -173,6 +178,46 @@ def is_openai_server_and_serving_model(
     "--temperature",
     cls=clickext.ConfigOption,
 )
+@click.option(
+    "--rag",
+    "rag_enabled",
+    default=False,
+    is_flag=True,
+)
+@click.option(
```

Comment:
pls see [here](https://github.com/instructlab/instructlab/blob/6cbcdabbe5b68a298ead63d89baccf97d63733dd/src/instructlab/cli/rag/ingest.py#L16) for using the `clickext` module to fetch the default from config